ext/Server/Bot.lua

```diff
@@ -32,6 +32,8 @@ function Bot:__init(player)
 	self._wayWaitTimer = 0;
 	self._wayWaitYawTimer = 0;
 	self._obstaceSequenceTimer = 0;
+	FirstShotRandomness = math.random (0, 2) / 10;
+	Config.botFirstShotDelay = Config.botFirstShotDelay + FirstShotRandomness;
```

Comment:
Don't write on the Config.vars, as they get used by the UI!! Use something like that instead:
`self._shotTimer = -MathUtils:GetRandom(Config.botFirstShotDelay, Config.botFirstShotDelay + 0.2);`